login.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./login.css" />
+  </head>
+  <body>
+    <form action="#!" id="main">
+      <h2>Login to your account</h2>
```

Comment:
Thumbs down for this. `h2` on the page without `h1` feels lonely. Fix that. Don't make `h2` sad!